src/DiagnosticErrorListener.ts

```diff
@@ -52,6 +55,18 @@ export class DiagnosticErrorListener implements ParserErrorListener {
 	}
 
 	@Override
+	syntaxError<T extends Token>(
+		/*@NotNull*/ recognizer: Recognizer<T, any>,
+		offendingSymbol: T | undefined,
+		line: number,
+		charPositionInLine: number,
+		/*@NotNull*/
+		msg: string,
+		e: RecognitionException | undefined): void
+	{
+	}
```

Comment:
Yes, ensures that use of this type will not trigger the new warning regarding weak types.